README.md

```diff
@@ -20,11 +20,36 @@ You can add this Git repository as a remote OPAM repository:
 
   ```
   opam repo add xs-opam https://github.com/xapi-project/xs-opam.git
-  opam pin add lwt 2.7.1
   ```
 
-Due to constraints missing from old libraries, pinning the `lwt` package
-is necessary for certain configurations.
+## Building packages
+
+To install a package it's enough to run
+
+  ```
+  opam install $PKGNAME
+  ```
+
+however, for development, it is often useful to clone the package sources
+and only build its dependencies, leaving the job to build the package
+and make changes to the developer.  This can be done as follows:
+
+  ```
+  opam install --deps-only $PKGNAME
+
+  # enter in the folder containing the cloned sources and run the
```

Comment:
I didn't want to write it at all... but if you copy paste you see what next on the terminal. It seemed to work on other snippets in other documentations